lib/translations_checker/checker.rb

```diff
@@ -14,6 +14,8 @@ class Checker
 
     using Indentation
 
+    SUPPORTED_FORMATS = %w(yml).freeze
```

Comment:
See my other comment.

```suggestion
    SUPPORTED_FORMATS = %w(.yml).freeze
```